### PR TITLE
Put custom code behind switch.

### PR DESCRIPTION
**Pull request by slowder**
@zenryan Puts the custom code behind a switch. Will allow for disabling via local_settings without a full deploy.

### CODE REVIEW

File: django/db/models/query.py [hunk 1]
```diff
@@ -62,7 +62,8 @@ def __init__(self, model=None, query=None, using=None, hints=None):
         self._prefetch_done = False
         self._known_related_objects = {}        # {rel_field, {pk: rel_obj}}
         # The following is a customization.
-        self.query.comment = self._custom_get_caller_info()
+        if getattr(settings, 'USE_MYSQL_COMMENTS', False):
```
Comment (jessecollier) [inline django/db/models/query.py]: this may not work as you expect unfortunately... have you tested this with our repo?

Comment (slowder) [inline django/db/models/query.py]: Yes, it works in `local_settings.py`

**Comment by zenryan** [comment]
looks fine to me

**Comment by jessecollier** [comment]
Remember, that in order to deploy this, we have to bump version number...

**Comment by zenryan** [comment]
yep, that should be a separate commit pre-release

**Pull request merged**